### PR TITLE
Adding README file

### PR DESCRIPTION
**Pull request by raynerpv2022**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+rearrange
```
Comment (randyyu-94) [inline README.md]: This is a test review.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
...
+this is  a new README file, it was not inside the origin repo
```
Comment (randyyu-94) [inline README.md:4]: This is another test review.

**Review by randyyu-94 — COMMENTED** [review]
This is a review containing two comments.

**Review by randyyu-94 — COMMENTED** [review]
This is a review containing two comments.

File: README.md (hunk 1 above)
Comment (raynerpv2022) [reply README.md]: practicing with test review

**Comment by raynerpv2022** [comment]
this bug  is fixed   :)

**Pull request closed without merging**